app/services/translations/translations.ts

```diff
@@ -167,5 +167,8 @@ export const translations = {
     inputRequired: {
       de: "Bitte treffen Sie eine Auswahl.",
     },
+    invalidHousenumber: {
+      de: "Bitte geben Sie eine gueltige Hausnummer ein.",
```

Comment:
I think you can use umlaut for the user facing string, no? 
```suggestion
      de: "Bitte geben Sie eine gültige Hausnummer ein.",
```

Reply:
excellent catch - AI slop ;)